pages.pt_PT/linux/pacman.md

```diff
@@ -0,0 +1,36 @@
+# pacman
+
+> Utilitário para gerir pacotes Arch Linux
+> Sub comandos como `pacman sync` tem a sua própria documentação
+> Mais informações em: <https://man.archlinux.org/man/pacman.8>.
+
+- Sincronizar e actualizar todos os pacotes:
```

Comment:
Use this mood please:
```suggestion
- Sincroniza e actualiza todos os pacotes:
```

Reply:
I would keep them in infinitive form as the remaining descriptions.